### PR TITLE
switch from random -> rand

### PR DESCRIPTION
**Pull request by cjkjellander**
This caused a warning and hence a compilation error on OTP 19.

### CODE REVIEW

File: src/ensq_channel.erl [hunk 1]
```diff
@@ -122,7 +122,7 @@ loop(State) ->
                              %% We don't want to ask for a propper new RC every time
                              %% this keeps the laod of the flow manager by guessing
```
Comment (elvisci) [inline src/ensq_channel.erl:123]: Line 123 is too long:                              %% this keeps the laod of the flow manager by guessing.

File: src/ensq_channel.erl [hunk 2]
```diff
@@ -122,7 +122,7 @@ loop(State) ->
                              %% We don't want to ask for a propper new RC every time
```
Comment (elvisci) [inline src/ensq_channel.erl:122]: Line 122 is too long:                              %% We don't want to ask for a propper new RC every time.

File: src/ensq_channel.erl [hunk 3]
```diff
@@ -122,7 +122,7 @@ loop(State) ->
...
                              %% this keeps the laod of the flow manager by guessing
                              %% we'll get the the same value back anyway.
-                             case random:uniform(10) of
+                             case rand:uniform(10) of
```
Comment (elvisci) [inline src/ensq_channel.erl:125]: The expression on line 125 and column 30 is nested beyond the maximum level of 3.

**Comment by suexcxine** [comment]
Since this has been fixed, we can close it.
Thank you.

**Pull request closed without merging**